osi_trafficcommand.proto

```diff
@@ -461,8 +480,181 @@ message SpeedAction
 
 }
 
+// \brief Longitudinal Distance Action
+//
+// The action tells the traffic participant to reach a longitudinal distance to 
+// another traffic participant. The longitudinal distance is defined as the distance
+// along the lane. 
+//
+// \note This action is aligned with LongitudinalDistanceAction of OpenSCENARIO 1.0
+// defining the reference traffic participanz and the distance.
+//
+
+message LongitudinalDistanceAction 
```

Comment:
@lemmer-fzi there are some identation bugs in this message.

Reply:
Fixed in commit ec066b0.